src/ManagedProcess.php

```diff
@@ -50,8 +51,7 @@ public function start( ProcessManager $manager, $data = [], $pid = null ) {
 		}
 
 		$this->parentProcess = new AsyncProcess( [
-			$phpBinaryPath, $scriptPath, $maintenancePath, $pid
-		] );
+			$phpBinaryPath, $scriptPath, $maintenancePath, $pid ] + $addInfo );
 		$input = new InputStream();
```

Comment:
Unfortunately I have already merged the "structured logging" PR. Could you please log all the arguments that are passed? Also the "input" and "output"

Reply:
eeyup sure... :raised_hands: